src/SFA.DAS.ApprenticeAccounts.Web/Pages/NotificationSettings.cshtml

```diff
@@ -0,0 +1,81 @@
+﻿@page
+@model NotificationSettingsModel
+@{
+    ViewData["Title"] = "Change your email notifications settings";
+}
+<form method="post">
+    <a href="@Model.BackLink" class="govuk-back-link">Return to home</a>
```

Comment:
There is a section we should use for breadcrumbs in the layout:
https://github.com/SkillsFundingAgency/das-apprentice-portal-web/blob/main/src/SFA.DAS.ApprenticePortal.SharedUi/Views/Shared/_Layout.cshtml#L49

You could use it something like:
`@section BreadCrumbs
{
    <a href="@Model.BackLink" class="govuk-back-link">Return to home</a>
}
`